pkg/apis/autoscaling/register.go

```diff
@@ -60,9 +60,11 @@ const (
 	//   autoscaling.knative.dev/metric: cpu
 	//   autoscaling.knative.dev/target: "75"   # target 75% cpu utilization
 	TargetAnnotationKey = GroupName + "/target"
-	// TargetMin is the minimum allowable target. Values less than
-	// zero don't make sense.
-	TargetMin = 1
+	// TargetMin is the minimum allowable target.
+	// This can be less than 1 due to the fact that with small container
+	// concurrencies and small target utilization values this can get
+	// below 1.
```

Comment:
You dropped the bit about values less than zero, I assume they still don't make sense?

Reply:
Yeah, negative targets are kind of obviously meaningless in this setting.